blog/docs/steering/cncf.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Knative accepted as a CNCF incubating project
+date: 2022-03-02
+---
+
+Hello Knative community,
+
+Today we are thrilled to announce that Knative has been [accepted into the Cloud Native Computing Foundation (CNCF) as an Incubating project!](https://www.cncf.io/blog/2022/03/02/knative-accepted-as-a-cncf-incubating-project) This is a significant milestone for our project, and we are excited to join so many successful cloud native open source projects. It is a massive testament to the outstanding contributors, and community Knative has - this achievement would not have been possible without you! Thank you :red_heart:
+
+Now that the project has been officially accepted, we continue the work of moving our project infrastructure and supporting tooling over to the [CNCF](https://www.cncf.io). Much of this was already underway as part of our previous work on open governance, and if you’d like to get involved and help out, any help is appreciated. Our goal is to minimize any potential disruptions to the project as this work is completed, and we will update you as things progress. Most of you won’t see any changes, but we are taking this opportunity to review the infrastructure and operations tooling we use for things like chat and meetings. We’re committed to using the tools that best serve our community and will be asking for your thoughts and feedback soon.
+
+We are very excited about this important step for the Knative community and hope to celebrate with you at [KnativeCon](https://events.linuxfoundation.org/knativecon-europe) this May. We also recognize that their attention is focused elsewhere for many in our global community. We wish our Ukrainian community members and contributors throughout Eastern Europe, safety for themselves and their families and peace soon.
```

Comment:
```suggestion
We are very excited about this important step for the Knative community and hope to celebrate with you at [KnativeCon](https://events.linuxfoundation.org/knativecon-europe) this May. We also recognize that many in our global community have their attention focused elsewhere. We wish our Ukrainian community members and contributors throughout Eastern Europe safety for themselves and their families and peace soon.
```